### PR TITLE
10722: Text color is not applied to all chiclets

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: src/chicletSlicer.ts [hunk 1]
```diff
@@ -868,6 +868,7 @@ module powerbi.extensibility.visual {
 
             labelTextSelection.style({
                 'font-size': PixelConverter.fromPoint(settings.slicerText.textSize),
+                'color': settings.slicerText.fontColor
```
Comment (ignatvilesov) [inline src/chicletSlicer.ts]: How did it work before?

File: test/visualTest.ts [hunk 2]
```diff
@@ -1004,17 +1004,46 @@ module powerbi.extensibility.visual.test {
...
+
+                    dataView.metadata.objects = {
+                        rows: {
+                            fontColor: getSolidColorStructuralObject(secondColor)
+                        }
+                    };
+
+                    visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                    let firstTextItem = visualBuilder.slicerTextElements.first();
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please specify a type of the variable.

File: src/chicletSlicer.ts (hunk 1 above)
Comment (savvinsergey) [inline src/chicletSlicer.ts]: It didn't work. Cause was CSS styles conflict

File: test/visualTest.ts (hunk 2 above)
Comment (savvinsergey) [inline test/visualTest.ts]: Done

**Pull request merged**